lib/surface/compiler/helpers.ex

```diff
@@ -102,11 +102,11 @@ defmodule Surface.Compiler.Helpers do
     assigns
   end
 
-  def to_meta(tree_meta, %CompileMeta{caller: caller, checks: checks}) do
+  def to_meta(tree_meta, %CompileMeta{caller: caller, checks: checks} = compile_meta) do
     %AST.Meta{
-      line: tree_meta.line,
-      column: tree_meta.column,
-      file: tree_meta.file,
+      line: tree_meta[:line] || compile_meta.line,
```

Comment:
is this change because the default sub block has empty meta? If that's the case, we could also update https://github.com/surface-ui/surface/blob/05d8cfbb1a8b2e70547ab550852faaab62ccebcc/lib/surface/compiler/parse_tree_translator.ex#L60 and https://github.com/surface-ui/surface/blob/05d8cfbb1a8b2e70547ab550852faaab62ccebcc/lib/surface/compiler/parse_tree_translator.ex#L56 to pass `to_meta(meta)` instead of `%{}`